packages/@aws-cdk/aws-lambda-destinations/test/integ.destinations.ts

```diff
@@ -0,0 +1,56 @@
+import lambda = require('@aws-cdk/aws-lambda');
+import sns = require('@aws-cdk/aws-sns');
+import sqs = require('@aws-cdk/aws-sqs');
+import { App, Construct, Duration, Stack, StackProps } from '@aws-cdk/core';
+import destinations = require('../lib');
+
+// After deploy, test with:
+// aws lambda invoke --function-name <deployed fn name> --invocation-type Event --payload '"OK"' response.json
+// aws lambda invoke --function-name <deployed fn name> --invocation-type Event --payload '"NOT OK"' response.json
+
+class TestStack extends Stack {
```

Comment:
We need to add a `DependsOn` on the alias because it's not CF-referenced in `AWS::Lambda::EventInvokeConfig` (only fn name and alias name are referenced). Will update code.

Reply:
Yep, that seems to be the problem. I got to the same conclusion.

Reply:
Alternatively we can `Fn.select(7, Fn.split(':', alias.ref))` to create the dependency

Reply:
Sounds good to me. I'd like an integration test along the lines of the code block I pasted above.

Reply:
done already